std_msgs/Header header
Robot[] robotList
